### ISSUE TITLE
data: clean up data on Zendo

### ISSUE THREAD

**Issue by krzysztof** [zenodo/zenodo#919]
Run the auditor task to find non-serializable record and fix them manually with FTFY.

Also manually check for similar issues with deposit and fix it as well.

**Comment by krzysztof** [comment]
Depends on #918 #917

**Comment by slint** [comment]
We should integrate into tests/checks what happens when general weird string input is provided for records using this list: https://github.com/minimaxir/big-list-of-naughty-strings

**Comment by krzysztof** [comment]
:+1: It could be as an extra unit test to Deposit API (https://github.com/zenodo/zenodo/blob/master/tests/unit/deposit/test_api_metadata.py) with a fixture of possible naughty inputs.

It's probably easy to implement and check, but this might be deserving of its own issue. I worry we might discover too much problems to fix within this sprint.

**Comment by slint** [comment]
Created #923 923

**Comment by slint** [comment]
@lnielsen @krzysztof I guess this could be closed, all the "cleaning" PRs have been merged and fixes have been applied on production. Maybe do a last check?

**Comment by krzysztof** [comment]
I guess this task is now done. The checks can be part of the auditor ticket #918
